packages/airnode-deployer/test/fixtures/index.ts

```diff
@@ -1 +0,0 @@
-export * from './config';
```

Comment:
I didn't even know we had these in deployer. Where did they come from?

Reply:
I think they were introduced when unit tests were added.